scraper/src/github-scraper/discussion.ts

```diff
@@ -37,54 +36,111 @@ query($org: String!, $cursor: String) {
                   }
                 }
                 createdAt
+                updatedAt
               }
             }
           }
         }
       }
     }
   }
-}
-`;
+}`;
 
-async function fetchDiscussionsForOrg(org: string, cursor = null) {
-  const variables = { org, cursor };
-  const response = await octokit.graphql.paginate(query, variables);
+async function fetchGitHubDiscussions(
+  org: string,
+  endDate: Date,
+  startDate: Date,
+  cursor = null,
+) {
+  const variables = {
+    org,
+    cursor,
+  };
+  for await (const response of octokit.graphql.paginate.iterator(
+    query,
+    variables,
+  )) {
+    const repositories = await response.organization.repositories.edges;
```

Comment:
Why await a non async property?

```suggestion
    const repositories = response.organization.repositories.edges;
```